tests/meta-mender-ci/recipes-mender/mender-monitor/mender-monitor_%.bbappend

```diff
@@ -0,0 +1,16 @@
+# Produce custom level logs when MENDER_CI_LOGLEVEL is enabled
+do_install_append_class-target_mender-image_mender-systemd() {
+    if ${@'true' if d.getVar('MENDER_CI_LOGLEVEL') else 'false'}; then
+        local logging_level=""
+        if [ "${MENDER_CI_LOGLEVEL}" = "debug" ]; then
```

Comment:
I would also check if the `MENDER_CI_LOGLEVEL` is among the: `FATAL TRACE ERROR DEBUG WARN INFO` otherwise fallback to one of them.

Reply:
Thanks. In practice this will only be used for debug in our CI, so I'll check for `DEBUG` and `TRACE` (to align it with mender-connect).

Reply:
Added support for `trace` :+1: